### DISCUSSION TITLE
Write e2e test for moving boulders

### DISCUSSION THREAD

**Discussion by Humorloos** [Humorloos/bouldern-app#422] [General]
Until now, I haven't been able to trigger the modify interaction from cypress. Closest I could get with https://github.com/cypress-io/cypress/issues/1624 was:
```
    cy.get('div.ol-viewport > div.ol-unselectable.ol-layers > .vector-layer > canvas')
        .trigger('pointerdown', {
          eventConstructor: 'PointerEvent',
          clientX: 50,
          clientY: 370,
          pointerId: 1,
          pointerType: 'touch',
          isPrimary: true,
          pressure: 1,
          composed: true,
          buttons: 1,
          isTrusted: true,
          width: 23,
        });
```
which also does not trigger the modify event though :(

**Comment by jaure96** [comment]
Hi @Humorloos 
Did you manage to pan the map??

I'm trying with this code... but nothing. I'm usin OpenLayers 6.10.0

```
cy.fixture('maritimeObjects').then((maritimeObjects) => {
    cy.wait(1000)

    cy.get(maritimeObjects.mapCanvas)
      .trigger('pointerdown', {
        x: 50,
        y: 370,
        pointerType: 'touch',
        pointerId: 1,
      })
      .trigger('pointermove', {
        x: 300,
        y: 200,
        pointerType: 'touch',
        pointerId: 1,
      })
      .trigger('pointerup', {
        x: 300,
        y: 200,
        pointerType: 'touch',
        pointerId: 1,
      })
  }) 
```
The fixture mapCanvas is this selector:
`"mapCanvas": "div.ol-map-container div.ol-viewport div.ol-unselectable.ol-layers div.ol-layer canvas"`

**Comment by Humorloos** [comment]
@jaure96 yes, panning works in my tests now. I wrote a helper function for this which you can find here: https://github.com/Humorloos/bouldern-app/blob/7707bdad1062901b78fb0282e0d8c05c1a88f7f4/frontend/cypress/support/functions.js#L169-L187

I'm using open layers 6.9.0, but I think this should also work with 6.10.0

**Reply by jaure96** [reply]
Humm... yeap! 
I had already seen how you were doing it and tried to replicate it in my project. But I didn't succeed.
Maybe it's the selector that I have wrong, because as far as I can see you are not selecting the open layers canvas.

**Reply by Humorloos** [reply]
I'm not selecting the canvas, no. I select #id_map-root, which is the id of my openlayers map's target element.
